src/cephadm/cephadm

```diff
@@ -1371,6 +1371,16 @@ class FileLock(object):
 
         return None
 
+    def delete(self) -> None:
+        """
+        Deletes the lock file.
+        """
+        if not self.is_locked:
+            if os.path.exists(self._lock_file):
+                os.remove(self._lock_file)
+        else:
+            logger.warning('Trying to delete a locked object %s ', self._lock_file)
```

Comment:
not sure if we really want to change things here. are you sure this doesn't create any race conditions? See `_release` a few lines below

Reply:
If we don't want to delete the lock then the alternative is just to leave it (this is the current behavior .. even after the cluster has been removed).  In the worst case we will have several .lock files that are not used.